src/utils/display/displayCanTakeApprenticeship.ts

```diff
@@ -0,0 +1,8 @@
+export const displayCanTakeApprenticeship = (num: number) => {
```

Comment:
Bardzo niebezpiecznie. To nie są enumy? Są. Więc typy powinny być enumowe i case-y tak samo!